app/javascript/blacklight/checkbox_submit.js

```diff
@@ -64,7 +68,7 @@ export default class CheckboxSubmit {
   }
 
   bookmarkCounter() {
```

Comment:
```suggestion
  bookmarkCounters() {
```

Could we rename the method name, since it could potentially be more than one counter?

Reply:
yes, you're right!